package.json

```diff
@@ -10,20 +10,23 @@
   "author": "Mozilla Add-ons Team",
   "license": "MPL-2.0",
   "dependencies": {
-    "@types/download": "6.2.4",
     "@types/yauzl": "2.9.1",
-    "body-parser": "1.19.0",
     "common-tags": "1.8.0",
-    "download": "8.0.0",
-    "express": "4.17.1",
     "first-chunk-stream": "3.0.0",
-    "safe-compare": "1.1.4",
     "strip-bom-stream": "4.0.0",
     "upath": "2.0.1",
     "yauzl": "2.10.0"
   },
+  "peerDependencies": {
+    "@types/download": "6.2.4",
+    "body-parser": "1.19.0",
+    "download": "8.0.0",
+    "express": "4.17.1",
+    "safe-compare": "1.1.4"
+  },
```

Comment:
it is a bit unfortunate that the peerDependencies are all being duplicated in the devDependencies, which I guess is for testing purpose, and from a quick look it seems that this is the common practice in this cases and so that's annoying but fine (especially given that renovatebot should be already helping us to be sure to update both at the same time)